src/amberclient/common/amber_client.py

```diff
@@ -129,10 +142,12 @@ def message_receiving_loop(self):
                                                (header.deviceType, header.deviceID))
 
                 except Exception as e:
+                    # FIXME: silent pass?
                     traceback.print_exc()
                     self.__logger.warn('Unknown error: %s' % str(e))
 
             except timeout:
+                # FIXME: silent pass?
```

Comment:
It is not silent pass. `timeout` is generate by non-blocking socket every 0.5 seconds if nothing is returned.